performance/benchmarks/cpus-1/summary.md

```diff
@@ -0,0 +1,106 @@
+# WSO2 Choreo Connect Performance Test Results (router CPUs = 1 , router concurrency level = 2)
+
+During each release, we execute various automated performance test scenarios and publish the results.
+
+| Test Scenarios | Description |
+| -------------- | ----------- |
+| Invoke an API deployed in the API Manager via Choreo Connect. | A secured API, which directly invokes the backend through Choreo Connect using JWT tokens considering different user counts and message payload sizes. |
+
+Our test client is [Apache JMeter](https://jmeter.apache.org/index.html). We test each scenario for a fixed duration of
+time (15 minutes). We split the test results into warmup (5 minutes) and measurement parts and use the measurement part (test results after 5 minutes) to compute the
+performance metrics. Below diagram shows the test setup.
+
+Test scenarios use a [Netty](https://netty.io/) based back-end service which echoes back any request
+posted to it.
+
+Below diagram shows the test setup.
+![picture](images/diagram.png)
+
+| Name                          | EC2 Instance Type | vCPU | Mem(GiB) |
+| ----------------------------- | ----------------- | ---- | -------- |
+| Apache JMeter Client          | c5.large          | 2    | 4        |
+| Apache JMeter Server 01       | c5.xlarge         | 4    | 8        |
+| Apache JMeter Server 02       | c5.xlarge         | 4    | 8        |
+| AWS EKS cluster (three nodes) | c5.xlarge         | 4    | 8        |
```

Comment:
Lets include specs of router, adapter, enforcer, netty, and nginx.

Reply:
Lets include these after line 48.

Reply:
Fixed with: 3e6d38b